.pylintrc

```diff
@@ -9,7 +9,9 @@ disable=
     cyclic-import,
     useless-object-inheritance,
     useless-import-alias,
-    useless-suppression
+    useless-suppression,
+    import-outside-toplevel,
+    wrong-import-order
```

Comment:
which part of code cause wrong-import-order?

Reply:
```
py38 run-test: commands[2] | pylint azdev --rcfile=.pylintrc -r n
************* Module azdev.__main__
azdev\__main__.py:19:8: C0415: Import outside toplevel (.__VERSION__) (import-outside-toplevel)
azdev\__main__.py:25:8: C0415: Import outside toplevel (azdev.commands.load_command_table) (import-outside-toplevel)
azdev\__main__.py:31:8: C0415: Import outside toplevel (azdev.params.load_arguments) (import-outside-toplevel)
************* Module azdev.operations.code_gen
azdev\operations\code_gen.py:92:4: C0415: Import outside toplevel (tempfile) (import-outside-toplevel)
azdev\operations\code_gen.py:93:4: C0415: Import outside toplevel (zipfile) (import-outside-toplevel)
azdev\operations\code_gen.py:126:4: C0415: Import outside toplevel (shutil) (import-outside-toplevel)
azdev\operations\code_gen.py:183:4: C0415: Import outside toplevel (jinja2.Environment, jinja2.PackageLoader) (import-outside-toplevel)
************* Module azdev.operations.performance
azdev\operations\performance.py:172:4: C0415: Import outside toplevel (multiprocessing) (import-outside-toplevel)
azdev\operations\performance.py:209:4: C0415: Import outside toplevel (signal) (import-outside-toplevel)
azdev\operations\performance.py:221:4: C0415: Import outside toplevel (math.sqrt) (import-outside-toplevel)
************* Module azdev.operations.pypi
azdev\operations\pypi.py:131:4: C0415: Import outside toplevel (tempfile) (import-outside-toplevel)
azdev\operations\pypi.py:132:4: C0415: Import outside toplevel (shutil) (import-outside-toplevel)
azdev\operations\pypi.py:199:4: C0415: Import outside toplevel (zipfile) (import-outside-toplevel)
azdev\operations\pypi.py:280:4: C0415: Import outside toplevel (difflib) (import-outside-toplevel)
azdev\operations\pypi.py:297:4: C0415: Import outside toplevel (filecmp) (import-outside-toplevel)
************* Module azdev.operations.resource
azdev\operations\resource.py:28:4: C0415: Import outside toplevel (datetime.datetime, datetime.timedelta) (import-outside-toplevel)
************* Module azdev.operations.setup
azdev\operations\setup.py:133:8: C0415: Import outside toplevel (platform) (import-outside-toplevel)
azdev\operations\setup.py:146:4: C0415: Import outside toplevel (glob.glob) (import-outside-toplevel)
azdev\operations\setup.py:147:4: C0415: Import outside toplevel (importlib.import_module) (import-outside-toplevel)
azdev\operations\setup.py:163:4: C0415: Import outside toplevel (knack.prompting.prompt_y_n, knack.prompting.prompt) (import-outside-toplevel)
************* Module azdev.operations.extensions.util
azdev\operations\extensions\util.py:72:4: C0415: Import outside toplevel (requests) (import-outside-toplevel)
************* Module azdev.operations.extensions
azdev\operations\extensions\__init__.py:84:4: C0415: Import outside toplevel (glob.glob) (import-outside-toplevel)
azdev\operations\extensions\__init__.py:105:4: C0415: Import outside toplevel (glob.glob) (import-outside-toplevel)
azdev\operations\extensions\__init__.py:138:4: C0415: Import outside toplevel (hashlib) (import-outside-toplevel)
azdev\operations\extensions\__init__.py:147:4: C0415: Import outside toplevel (azdev.operations.setup._check_repo) (import-outside-toplevel)
azdev\operations\extensions\__init__.py:187:4: C0415: Import outside toplevel (re) (import-outside-toplevel)
azdev\operations\extensions\__init__.py:188:4: C0415: Import outside toplevel (tempfile) (import-outside-toplevel)
azdev\operations\extensions\__init__.py:190:4: C0415: Import outside toplevel (util.get_ext_metadata, util.get_whl_from_url) (import-outside-toplevel)
azdev\operations\extensions\__init__.py:275:4: C0415: Import outside toplevel (azure.storage.blob.BlockBlobService) (import-outside-toplevel)
************* Module azdev.operations.help
azdev\operations\help\__init__.py:304:4: C0415: Import outside toplevel (requests) (import-outside-toplevel)
************* Module azdev.operations.linter.linter
azdev\operations\linter\linter.py:157:8: C0415: Import outside toplevel (knack.deprecation.Deprecated) (import-outside-toplevel)
azdev\operations\linter\linter.py:278:8: C0415: Import outside toplevel (colorama.Fore) (import-outside-toplevel)
************* Module azdev.operations.linter.util
azdev\operations\linter\util.py:89:4: C0415: Import outside toplevel (azure.cli.core.commands.ExtensionCommandSource) (import-outside-toplevel)
************* Module azdev.operations.linter
azdev\operations\linter\__init__.py:34:4: C0415: Import outside toplevel (azure.cli.core.get_default_cli) (import-outside-toplevel)
azdev\operations\linter\__init__.py:35:4: C0415: Import outside toplevel (azure.cli.core.file_util.get_all_help, azure.cli.core.file_util.create_invoker_and_load_cmds_and_args) (import-outside-toplevel)
************* Module azdev.operations.tests
azdev\operations\tests\__init__.py:122:4: C0415: Import outside toplevel (azdev.utilities.diff_branches, azdev.utilities.extract_module_name) (import-outside-toplevel)
azdev\operations\tests\__init__.py:123:4: C0415: Import outside toplevel (azdev.utilities.git_util.summarize_changed_mods) (import-outside-toplevel)
azdev\operations\tests\__init__.py:266:8: C0415: Import outside toplevel (azdev.utilities.extract_module_name) (import-outside-toplevel)
************* Module azdev.utilities.command
azdev\utilities\command.py:24:4: C0415: Import outside toplevel (azdev.utilities.IS_WINDOWS) (import-outside-toplevel)
azdev\utilities\command.py:40:4: C0415: Import outside toplevel (azdev.utilities.IS_WINDOWS, azdev.utilities.display) (import-outside-toplevel)
azdev\utilities\command.py:70:4: C0415: Import outside toplevel (azdev.utilities.get_env_path) (import-outside-toplevel)
************* Module azdev.utilities.config
azdev\utilities\config.py:22:4: C0415: Import outside toplevel (azdev.utilities.get_env_path) (import-outside-toplevel)
************* Module azdev.utilities.git_util
azdev\utilities\git_util.py:47:4: C0415: Import outside toplevel (azdev.utilities.extract_module_name) (import-outside-toplevel)
azdev\utilities\git_util.py:64:8: C0415: Import outside toplevel (git) (import-outside-toplevel)
azdev\utilities\git_util.py:65:8: C0415: Import outside toplevel (git.exc) (import-outside-toplevel)
azdev\utilities\git_util.py:66:8: C0415: Import outside toplevel (gitdb) (import-outside-toplevel)
azdev\utilities\git_util.py:70:4: C0415: Import outside toplevel (git.Repo) (import-outside-toplevel)
************* Module azdev.utilities.path
azdev\utilities\path.py:17:4: C0415: Import outside toplevel (re) (import-outside-toplevel)
azdev\utilities\path.py:60:4: C0415: Import outside toplevel (configparser.NoSectionError) (import-outside-toplevel)
azdev\utilities\path.py:61:4: C0415: Import outside toplevel (config.get_azdev_config) (import-outside-toplevel)
azdev\utilities\path.py:73:4: C0415: Import outside toplevel (configparser.NoSectionError) (import-outside-toplevel)
azdev\utilities\path.py:74:4: C0415: Import outside toplevel (config.get_azdev_config) (import-outside-toplevel)
azdev\utilities\path.py:109:4: C0415: Import outside toplevel (errno) (import-outside-toplevel)
azdev\utilities\path.py:122:4: C0415: Import outside toplevel (azure.cli.core.extension.EXTENSIONS_DIR) (import-outside-toplevel)
azdev\utilities\path.py:193:4: C0415: Import outside toplevel (azure.cli.core.extension.EXTENSIONS_DIR) (import-outside-toplevel)
************* Module azdev.utilities.testing
azdev\utilities\testing.py:9:4: C0415: Import outside toplevel (azdev.__main__.main) (import-outside-toplevel)
azdev\utilities\testing.py:10:4: C0415: Import outside toplevel (sys) (import-outside-toplevel)
************* Module azdev.utilities.tools
azdev\utilities\tools.py:11:4: C0415: Import outside toplevel (azdev.utilities.get_env_path) (import-outside-toplevel)
azdev\utilities\tools.py:20:8: C0415: Import outside toplevel (azure.cli.core) (import-outside-toplevel)

-----------------------------------
Your code has been rated at 9.81/10
```